devices/sonoff/snzb-06p.json

```diff
@@ -0,0 +1,126 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "SONOFF",
+  "modelid": "SNZB-06P",
+  "vendor": "Sonoff",
+  "product": "SNZB-06P",
+  "sleeper": false,
+  "status": "Gold",
+  "path": "/devices/snzb-06p.json",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
```

Comment:
You might want to consider adding `config/delay` as well, as the device allows setting the interval when it changes from occupied to unoccupied via attribute 0x0020.

An example would be the Philips motion sensor:
https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/philips/sml001_motion_sensor.json